### PR TITLE
hotfix: update ingress resource configuration

### PR DESCRIPTION
**Pull request by danielogen**
# HOTFIX: update ingress resource configuration

**_WHAT DOES THIS PR DO?_**

Update Nginx ingress resource configuration on staging to match the latest standards

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
N/A
**_HOW DO I TEST OUT THIS PR?_**

- Already working on staging

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

-  All staging endpoints

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

N/A

**_ARE THERE ANY RELATED PRs?_**

N/A

### CODE REVIEW

File: k8s/nginx-ingress/stage-platform-ingress-resource.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
```
Comment (NoahNsimbe) [inline k8s/nginx-ingress/stage-platform-ingress-resource.yaml:1]: Thanks @danielogen, This had to be done after upgrading the Kubernetes cluster cc @123MwanjeMike

**Review by NoahNsimbe — APPROVED** [review]
LGTM.

**Comment by NoahNsimbe** [comment]
@123MwanjeMike you might want to check whether the configurations are still valid for other Kubernetes resources after upgrading the cluster.

File: k8s/nginx-ingress/stage-platform-ingress-resource.yaml (hunk 1 above)
Comment (123MwanjeMike) [reply k8s/nginx-ingress/stage-platform-ingress-resource.yaml:1]: Yes @NoahNsimbe.
This was done so all is well

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @danielogen 
This looks good to me.
In addition, I had also updated the production cluster so I think a similar change is needed for both the production api and platform ingresses as well. Do you have any changes being made on this currently? I would like to jump into it soon.

**Comment by Baalmart** [comment]
> Thanks @danielogen This looks good to me. In addition, I had also updated the production cluster so I think a similar change is needed for both the production api and platform ingresses as well. Do you have any changes being made on this currently? I would like to jump into it soon.

I can notice some pending comments here. About to do the merge duty but hope that all is well? @123MwanjeMike @danielogen

**Comment by 123MwanjeMike** [comment]
> @123MwanjeMike you might want to check whether the configurations are still valid for other Kubernetes resources after upgrading the cluster.

Yes @NoahNsimbe.
The new update will have to apply to all those other configurations.

**Comment by 123MwanjeMike** [comment]
> > Thanks @danielogen This looks good to me. In addition, I had also updated the production cluster so I think a similar change is needed for both the production api and platform ingresses as well. Do you have any changes being made on this currently? I would like to jump into it soon.
> 
> I can notice some pending comments here. About to do the merge duty but hope that all is well? @123MwanjeMike @danielogen

Thanks @Baalmart. All's fine.

**Pull request merged**